portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java

```diff
@@ -56,16 +59,23 @@ public static ProcessConfig createProcessConfig(Class<?>... classes) {
 
 		File[] files = _listClassPathFiles(classes);
 
-		StringBundler classpathSB = new StringBundler(files.length * 2);
+		if (files.length == 0) {
```

Comment:
@julschong The changes in this method are not related to the commit message of this pull, please merge this part of change in to last commit.

Reply:
@julschong Please note one thing, that the throwing IllegalStateException is actually a change to the old code. If  you merge the code into last commit, you should not merge in this part. This should be separate commit.